README.md

```diff
@@ -1,25 +0,0 @@
-# integration-adaptor-111
```

Comment:
You shouldn't delete this one

Reply:
I have moved the Readme to test-scripts, like it is in nhais. I believe we need to README.md - general one and one for release